marimo/_plugins/ui/_impl/table.py

```diff
@@ -93,10 +93,15 @@ class SearchTableArgs:
     limit: Optional[int] = None
 
 
+CellStyle = dict[str, Any]
```

Comment:
could `Any` be a `dict[string, Any]`?

Reply:
It is nested structure:
- first dict contains the rowIds,
- next level the columnName
- last level the style dict

Reply:
Sure thing!